src/utility/STM32Cube_FW/app_conf.h

```diff
@@ -0,0 +1,539 @@
+/* USER CODE BEGIN Header */
+/**
+ ******************************************************************************
+  * File Name          : app_conf.h
```

Comment:
app_conf.h needs to be adapted to the application, basically it must be cleaned-up of all useless information.
From which cube application did you copy this ap_conf.h version? 
I think the one from transparent mode application is maybe the one that is closest to arduino library case

Reply:
app_conf.h  from stm32cube example (transparent_mode) is reduced to include the required defined constants only